pkgs/applications/misc/polybar/default.nix

```diff
@@ -39,6 +39,12 @@ stdenv.mkDerivation rec {
       maintainers = [ maintainers.afldcr ];
       platforms = platforms.unix;
     };
```

Comment:
Can we put a comment above this to mark that this patch should be removed at the next release? It makes it a bit easier on myself as I can't test locally at the moment-- my Linux box is out of commission.